<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="1,0" columnstretch="0">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="toolsTab">
      <attribute name="title">
       <string>Tools</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,1" columnstretch="0,1,0,0">
       <item row="1" column="0">
        <widget class="QLabel" name="pkcsEngineLibLabel">
         <property name="text">
          <string>PKCS#11 engine library</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="pkcsEngineLibAuto">
         <property name="text">
          <string>auto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/buttons/refresh</normaloff>:/buttons/refresh</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="pkcsEngineLibValue">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="toolsVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="pkcsToolBinLabel">
         <property name="text">
          <string>PKCS#11 tool</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="pkcsEngineLibBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="pkcsToolBinValue">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="pkcsToolBinBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="pkcsToolBinAuto">
         <property name="text">
          <string>auto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/buttons/refresh</normaloff>:/buttons/refresh</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="opensslBinLabel">
         <property name="text">
          <string>OpenSSL custom</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="opensslBinValue">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QToolButton" name="opensslBinBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QToolButton" name="opensslBinAuto">
         <property name="text">
          <string>auto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/buttons/refresh</normaloff>:/buttons/refresh</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="smartcardTab">
      <attribute name="title">
       <string>Smart Card</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="smartcardProducerLabel">
         <property name="text">
          <string>Producer</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="smartcardProducerAuto">
         <property name="text">
          <string>auto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/buttons/refresh</normaloff>:/buttons/refresh</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="smartcardLibLabel">
         <property name="text">
          <string>Library</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QToolButton" name="smartcardLibBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="smartcardProducerValue"/>
       </item>
       <item row="1" column="3">
        <widget class="QToolButton" name="smartcardLibAuto">
         <property name="text">
          <string>auto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/buttons/refresh</normaloff>:/buttons/refresh</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="smartcardLibValue">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="smartcardVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="certificateTab">
      <attribute name="title">
       <string>Certificate</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,1,0,0">
       <item row="0" column="0">
        <widget class="QLabel" name="certificateIdLabel">
         <property name="text">
          <string>Certificate ID</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="certificateIdValue"/>
       </item>
       <item row="0" column="2">
        <widget class="QToolButton" name="certificateIdReload">
         <property name="text">
          <string>auto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/buttons/reload</normaloff>:/buttons/reload</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QToolButton" name="certificateIdAuto">
         <property name="text">
          <string>auto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/buttons/refresh</normaloff>:/buttons/refresh</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="certificateVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
